Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -163,6 +202,21 @@ def parse?_errorContext (line : String) : Option ErrorContext := Id.run do
         | "ERR_SEM" => some (StyleError.semicolon)
         | "ERR_TWS" => some (StyleError.trailingWhitespace)
         | "ERR_WIN" => some (StyleError.windowsLineEnding)
+        | "ERR_UNICODE_VARIANT" => do
+          match (← errorMessage[0]?) with
```

Comment:
Relying on word position is the only part that looks a little fragile to me, but this is also already used by the linter anyway.

Could you add a test that specifically flags any change in words of the error message?

Reply:
Parsing the error message is brittle, but okay enough: please add a test which covers the current wording, though - to make sure to catch a wording change.